tools/cp-caps/README.md

```diff
@@ -123,3 +124,13 @@ diag gpio get 2 ------------------------------------------ NotSupported
 diag gpio set 2 0 ---------------------------------------- NotSupported
 diag gpio set 2 1 ---------------------------------------- NotSupported
 ```
+
+### Test Data Poll
+
+The parameter `-p` or `--data-poll` starts to test whether the RCP supports data poll.
+
+```bash
+$ DUT_ADB_USB=1269UCKFZTAM95OR REF_CLI_SERIAL=/dev/ttyACM0 python3 ./tools/cp-caps/rcp_caps_test.py -p
+Data Poll Receiver --------------------------------------- OK
```

Comment:
I think any FTD device should be able to act as a parent and therefore should be capable of processing data polls, so I guess this would be equivalent to `is this device FTD`?